services/app-api/handlers/form-templates/post/obtainFormTemplate.js

```diff
@@ -9,17 +10,21 @@ import dynamoDb from "../../../libs/dynamodb-lib";
 export const main = handler(async (event, context) => {
   // *** if this invocation is a pre-warm, do nothing and return
   if (event.source === "serverless-plugin-warmup") {
-    console.log("Warmed up!");
     return null;
```

Comment:
This log doesn't need to go and is mildly helpful when figuring out when lambdas were triggered and why